src/renderer/features/assets/AssetsPortfolioView/model/portfolio-model.ts

```diff
@@ -21,97 +23,54 @@ const $hideZeroBalances = restore(hideZeroBalancesChanged, false);
 const $accounts = restore<Account[]>(accountsChanged, []);
 const $activeView = restore<AssetsListView | null>(activeViewChanged, null);
 const $query = restore<string>(queryChanged, '');
-const $tokens = createStore<AssetByChains[]>([]);
-const $activeTokens = createStore<AssetByChains[]>([]);
-const $activeTokensWithBalance = createStore<AssetByChains[]>([]);
-const $filteredTokens = createStore<AssetByChains[]>([]);
-const $sortedTokens = createStore<AssetByChains[]>([]);
-
-const $tokensPopulated = createStore(false).on(once($sortedTokens.updates), () => true);
 
-type UpdateTokenParams = {
-  activeWallet?: Wallet;
-  chains: Record<ChainId, Chain>;
-};
-
-const getUpdatedTokensFx = createEffect(({ activeWallet, chains }: UpdateTokenParams): AssetByChains[] => {
-  const tokens = tokensService.getTokensData();
-  const updatedTokens: AssetByChains[] = [];
-
-  for (const token of tokens) {
-    const filteredChains = token.chains.filter((chain) => {
-      return activeWallet?.accounts.some((account) => {
-        return (
-          accountUtils.isNonBaseVaultAccount(account, activeWallet) &&
-          accountUtils.isChainAndCryptoMatch(account, chains[chain.chainId])
-        );
-      });
-    });
+const $basicTokens = createStore(tokensService.getTokensData());
```

Comment:
I am not clearly understand after first view what `basicTokens` means. Maybe call it `defaultTokens`?